.github/workflows/ci.yml

```diff
@@ -46,6 +46,9 @@ jobs:
       - name: Run unit tests
         run: composer test
 
+      - name: Run static analysis
+        run: composer phpstan
```

Comment:
What do you think about running PHPStan before running composer test?

Reply:
Since we use `yoast/phpunit-polyfills`, PHPUnit will not be available until tests are run.